third_party/proto/cosmos_proto/cosmos.proto

```diff
@@ -3,14 +3,14 @@ package cosmos_proto;
 
 import "google/protobuf/descriptor.proto";
 
-option go_package = "github.com/regen-network/cosmos-proto";
+option go_package = "github.com/cosmos/cosmos-proto";
 
 extend google.protobuf.MessageOptions {
-    string interface_type = 93001;
-
-    string implements_interface = 93002;
+    string implements_interface = 93001;
```

Comment:
Let's copy-paste the exact same file as https://github.com/cosmos/cosmos-proto/blob/main/cosmos.proto, with the proto comments

Reply:
Done